.github/workflows/ci.yml

```diff
@@ -0,0 +1,60 @@
+name: CI
+
+on:
```

Comment:
I think you'll need a `tags` entry here, like:
https://github.com/heroku/stack-images/blob/c151f87271e02aa7d8524cb46740e6a94355f771/.github/workflows/ci.yml#L8-L9